automation/src/test/resources/reference.conf

```diff
@@ -8,3 +8,5 @@ leonardo.currentWelderImage = "us.gcr.io/broad-dsp-gcr-public/welder-server:ed93
 leonardo.bioconductorImageUrl = "us.gcr.io/broad-dsp-gcr-public/terra-jupyter-bioconductor:0.0.9"
 leonardo.rstudioBaseImageUrl = "us.gcr.io/anvil-gcr-public/anvil-rstudio-base:0.0.2"
 leonardo.rstudioBioconductorImageUrl = "us.gcr.io/anvil-gcr-public/anvil-rstudio-bioconductor:0.0.3"
+leonardo.pubsubGoogleProject = "broad-dsde-dev"
+leonardo.topicName = "leonardo-pubsub-test" //each fiab will have a unique topic name sourced from leonardo.conf, this is largely irrelevant but needed to initializie some objects
```

Comment:
this comment is very important to note - all fiabs will have a unique topic based on the timestamp at which they are created, to prevent interference. The tests will also clean-up the topics after themselves.

Reply:
I updated the comment to be more clear, this topic is used in an auto test to ensure the subscriber can appropriately connect, but it is never published/read from. The unique topic is actually used in tests. 

So an error will be thrown if it is not present.